examples/cassandra/README.md

```diff
@@ -57,103 +71,51 @@ command line tool somewhere in your path.  Please see the
 for installation instructions for your platform.
```

Comment:
With the docs move this link is now just a reference to the new location.  ```[getting started guides](../../docs/getting-started-guides/)```.  I wonder if there is as better way of handling this.  As a reader I'd prefer the link to point directly at the new docs.